### ISSUE TITLE
paladin tank bouclier de piété + sceau d'autorité

### ISSUE THREAD

**Issue by MissZay** [way-of-elendil/3.3.5#1591]
**Description**
La compétences de paladin bouclier de piété n'active pas le sceau d'autorité

**Étapes de reproduction**
frapper une cible avec bouclier de piété et sceau d'autorité actif

**Comportement attendu**
le sceau d'autorité devrait se déclencher sur toute attaque ou compétence de mélée qui ne touche qu'une cible

**Captures d’écran / Vidéos**
https://www.youtube.com/watch?v=5__RAlbl8l8
le nombre de ticks de sceau d'autorité est toujours égal au nombre d'auto attaques même en utilisant bouclier de piété

**Ressources additionnelles**
https://woehead.way-of-elendil.fr/?spell=20375
https://woehead.way-of-elendil.fr/?spell=61411
https://www.wowhead.com/forums/topic/seal-of-command-good-for-tanking-143068
2ème post "The best part about it is that it now procs off Shield of Righteousness and Judgement"

**Comment by Wilpok** [comment]
Pour preuve supplémentaires  

https://youtu.be/IWla-y72Glo?si=rf5ILGKHYe2mqyzO

**Comment by MissZay** [comment]
Je ne l'avais pas mis dans le 1er post car j'ai trouvé des informations contradictoires mais sur la vidéo on voit bien à 0:29, 0:44, 1:15, 1:27, 1:33 que le marteau du vertueux active aussi le sceau d'autorité, sur woe ce n'est pas le cas non plus

**Comment by Wilpok** [comment]
D’ailleurs c’est bizarre vu que le vertueux proc bien le vengeance en pve de mémoire

**Comment by MissZay** [comment]
C'est bien le cas oui mais il fonctionne comme la tempête divine; chaque cible n'active le sceau d'autorité qu'une fois, pas trois
Dans la vidéo à 0:29 il y a 6 ticks de sceau d'autorité, 3 sur l'auto attaque et 1 pour chaque cible touchée par le marteau

**Comment by Wilpok** [comment]
Yes pour ça que je l’ai mis, vidéo de tlk « vanilla » qui corrobore tes dires. C’est juste tjrs mieux d’avoir des preuves concrètes et factuelles . Sachant qu’on peut retrouver tout et n’importe quoi dans les commentaires wowhead. Il est toujours plus pertinent d’étoffer avec des vidéos de t’a 14 ans ( pour tes prochains tickets si t’en fais)

**Comment by MissZay** [comment]
je préfère aussi les vidéos de l'époque aux commentaires souvent foireux mais j'avais rien trouvé en l'occurrence
